content/webapp/app.js

```diff
@@ -60,6 +60,7 @@ module.exports = app.prepare().then(async () => {
   route('/pages/:id', '/page', router, app);
 
   route('/opening-times', '/opening-times', router, app);
+  route('/opening-timesv2', '/opening-timesv2', router, app);
```

Comment:
When doing this, we need to figure a way to have these not index, which is why I regretted `worksv2`.
Do we feel this is easier than adding a toggle to `/opening-times`?